### PR TITLE
NavPi project page 

### PR DESCRIPTION
**Pull request by zanuka**
- [x] images approved
- [x] content approved
- [x] links checked
- [x] feedback incorporated

### CODE REVIEW

File: content/projects/nav-pi/index.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ lastmod: 2018-03-29T14:32:01+13:00
 draft: false
```
Comment (matt-auckland) [inline content/projects/nav-pi/index.md]: Page title should be NavPi not Nav Pi

File: content/projects/nav-pi/index.md [hunk 2]
```diff
@@ -14,14 +16,69 @@ show_title_text: false
...
+    subtitle="This software turns your Raspberry Pi into a low cost and convenient way to help maintain the network."
+    fontColor="#FFF"
+    subtitleFontColor="#FFF"
+    bgColor="#7d5ab5"
+    linkBgColor="#523971"
+    gradient=""
+    feature1Img="/images/icons/nav_purple-pi-large.png"
+    feature1Title="NavPi v1.0.8"
+    feature1Link="https://s3-ap-southeast-1.amazonaws.com/org.navcoin.navpi/navpi_1.0.8.img.torrent"
+    feature1ExtraSubtitle="Torrent Magnet"
```
Comment (matt-auckland) [inline content/projects/nav-pi/index.md]: These links don't seem to be coming through on the page, might be a typo in the var name

**Comment by zanuka** [comment]
cool @matt-auckland thx for quick review, fixed the links and title... more progress tomorrow

File: content/projects/nav-pi/index.md (hunk 1 above)
Comment (zanuka) [reply content/projects/nav-pi/index.md]: fixed @matt-auckland

**Pull request merged**